src/RA_Dlg_GameTitle.cpp

```diff
@@ -113,7 +113,7 @@ INT_PTR Dlg_GameTitle::GameTitleProc( HWND hDlg, UINT uMsg, WPARAM wParam, LPARA
 			{
 				//	Existing title
 				ASSERT( m_aGameTitles.find( std::string( sSelectedTitle ) ) != m_aGameTitles.end() );
-				nGameID = m_aGameTitles[ std::string( Narrow(sSelectedTitle) ) ];
+				nGameID = m_aGameTitles[ std::string(_RA Narrow(sSelectedTitle) ) ];
```

Comment:
I don't like hiding the namespace behind a `#define`. Suggest explicit `ra::Narrow`

Reply:
ok

Reply:
it's done